testcases/testcase_blood_flow_model.py

```diff
@@ -55,7 +58,7 @@
         "csv_diameter": "D", "csv_length": "L",
         "csv_edgelist_v1": "n1", "csv_edgelist_v2": "n2",
         "csv_coord_x": "x", "csv_coord_y": "y", "csv_coord_z": "z",
-        "csv_boundary_vs": "nodeId", "csv_boundary_type": "boundaryType", "csv_boundary_value": "boundaryValue",
+        "csv_boundary_vs": "nodeID", "csv_boundary_type": "boundaryType", "csv_boundary_value": "p",
```

Comment:
Please leave it as before.

Reply:
it doesn't work as before

Reply:
Same here. In the main branch, we would like to have the default parameters. It's just a label in a CSV file, it could be anything. Could you please just change these values to their initial values? Users can change them for their simulation settings.